algorithms/src/std_algorithms/impl/Kokkos_RandomAccessIterator.hpp

```diff
@@ -54,17 +54,18 @@ class RandomAccessIterator< ::Kokkos::View<DataType, Args...> > {
   KOKKOS_DEFAULTED_FUNCTION RandomAccessIterator() = default;
 
   explicit KOKKOS_FUNCTION RandomAccessIterator(const view_type view)
-      : m_view(view) {}
+      : m_data(view.data()), m_stride(view.stride_0()) {}
```

Comment:
I feel like you could still have the `m_view` member, and use `m_view.data()` when you need it. Right ?

Reply:
Not sure. Depends on what the limiter is.

Reply:
if the goal of this PR is to move towards using the pointer and not the view, i agree on NOT copying the view.